Wikipedia/Code/WMFAppViewController.m

```diff
@@ -1215,9 +1215,21 @@ - (BOOL)processUserActivity:(NSUserActivity *)activity animated:(BOOL)animated c
             [self.currentTabNavigationController popToRootViewControllerAnimated:animated];
             NSURL *articleURL = activity.wmf_linkURL;
             if (articleURL) {
```

Comment:
(Optional)

This may be tricky to fix, but I would prefer if we removed the whole `WMFArticleURL=` part of these places deep links:

```
wikipedia://places?WMFArticleURL=https://en.wikipedia.org/wiki?latitude=52.3547498&longitude=4.8339215
wikipedia://places?WMFArticleURL=https://en.wikipedia.org/wiki?place=Berlin
```

It is unnecessary information, and indicates we are searching against the subdomain wiki (`en.wikipedia.org`), when in reality the app searches against the primary app language wiki in Settings.

If you could find a way to get these two links to work instead:

```
wikipedia://places?latitude=52.3547498&longitude=4.8339215
wikipedia://places?place=Berlin
```

I think that would feel better.